### PR TITLE
Cms/382

### PR DESCRIPTION
**Pull request by danieljust**
Добавлен вывод страницы регистрации на localhost:3000/registration

### CODE REVIEW

File: backend/public/css/login.css [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+}
+
+body {
+    background-color: #ef6b50;
+}
+
+.t-loginpage {
+    max-width: 470px;
+    margin: 0 auto;
+    padding: 0 0;
```
Comment (mvahrameev) [inline backend/public/css/login.css]: Just `padding: 0;`

File: backend/public/css/login.css [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+
+.t-loginpage-wrapper {
+    display: table-cell;
+    margin: 24px;
+    white-space: normal;
+    vertical-align: middle;
+    min-width: 450px;
+}
+
+.t-loginform {
```
Comment (mvahrameev) [inline backend/public/css/login.css]: I can't find where this class is used

File: backend/public/css/login.css [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+    color: #fff;
+    text-decoration: none;
+    margin: 0 6px;
+    font-weight: 400;
+}
+
+.t-btn {
+    border-radius: 100px;
+    background-color: #ffe8c7;
+    padding: 10px 50px 10px;
```
Comment (IgorZiegler) [inline backend/public/css/login.css]: 10 push-ups for every extra pixel `padding: 10px 50px;`

File: backend/public/css/login.css [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+        margin: auto;
+    }
+
+    .t-loginpage-wrapper {
+        min-width: 180px;
+    }
+
+    .t-loginform {
+        max-width: 300px;
+        padding: 5px 10px 5px;
```
Comment (IgorZiegler) [inline backend/public/css/login.css]: + 50 push-ups

File: backend/public/css/login.css [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+    .t-links {
+        padding-top: 60px;
+    }
+
+    .t-loginform-item {
+        padding-top: 60px;
+    }
+
+    .t-btn {
+        padding: 10px 100px 10px;
```
Comment (IgorZiegler) [inline backend/public/css/login.css]: `padding: 10px 100px;`

File: backend/public/css/registration.css [hunk 6]
```diff
@@ -0,0 +1,202 @@
...
+    display: table-cell;
+    margin: 24px;
+    white-space: normal;
+    vertical-align: middle;
+}
+
+.regform-btn {
+    position: relative;
+    padding: 10px 67px 11px;
+    margin: 50px 0 0 0;
```
Comment (IgorZiegler) [inline backend/public/css/registration.css]: `margin: 50px 0 0;`

File: backend/public/css/registration.css [hunk 7]
```diff
@@ -0,0 +1,202 @@
...
+.regform {
+    background-color: rgba(0, 0, 0, 0.95);
+    padding: 30px 55px 37px;
+    border-radius: 8px;
+    box-sizing: border-box;
+    overflow: hidden;
+}
+
+.regform__item {
+    padding: 0 0 0 0;
```
Comment (IgorZiegler) [inline backend/public/css/registration.css]: `padding: 0 0 0 0;` - 10 push-ups for every extra 0

File: backend/public/css/registration.css [hunk 8]
```diff
@@ -0,0 +1,202 @@
...
+    margin: 29px 0 0 0;
```
Comment (IgorZiegler) [inline backend/public/css/registration.css]: `margin: 29px 0 0;`

File: backend/public/css/registration.css [hunk 9]
```diff
@@ -0,0 +1,202 @@
...
+    color: #fff;
+    text-align: center;
+    width: 100%;
+    height: 53px;
+    box-sizing: border-box;
+}
+
+.regform-input_custom {
+    padding: 0 10px;
+    margin: 29px 0 0 0;
```
Comment (IgorZiegler) [inline backend/public/css/registration.css]: As above

File: backend/public/css/registration.css [hunk 10]
```diff
@@ -0,0 +1,202 @@
...
+    height: 53px;
+    box-sizing: border-box;
+}
+
+.regform-input_custom:hover {
+    border-bottom: 1px solid #fff;
+}
+
+.regform-link__terms {
+    padding: 30px 0 0 0;
```
Comment (IgorZiegler) [inline backend/public/css/registration.css]: As above

File: backend/public/css/registration.css [hunk 11]
```diff
@@ -0,0 +1,202 @@
...
+        margin: auto;
+    }
+
+    .regpage-wrapper {
+        min-width: 180px;
+    }
+
+    .regform {
+        max-width: 300px;
+        padding: 5px 10px 5px;
```
Comment (IgorZiegler) [inline backend/public/css/registration.css]: As above

File: backend/public/css/registration.css [hunk 12]
```diff
@@ -0,0 +1,202 @@
...
+        height: 100%;
+        width: 300px;
+        text-align: center;
+    }
+
+    .regform-btn {
+        padding: 10px 30px 10px;
```
Comment (IgorZiegler) [inline backend/public/css/registration.css]: As above

File: backend/package.json [hunk 13]
```diff
@@ -4,12 +4,16 @@
   "description": "",
   "main": "app.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "nodemon app.js"
```
Comment (maxmalov) [inline backend/package.json]: I think I'd saw it in other PR. Again, consider delivering common features in a separate PRs. This PR wouldn't pass a review if will contain the same changes as others.

**Comment by maxmalov** [comment]
Review the commit history of this branch. It contains commits which don't belong to master or you. That means you've branched from the wrong place

**Pull request closed without merging**